config/locales/en.yml

```diff
@@ -320,6 +320,10 @@ en:
             show: Confirm trainee’s course details
           course_details:
             edit: Review course
+        forbidden_deletes:
+          show: Delete Forbidden
```

Comment:
Should it be `Delete forbidden`? The capitalisation here looks out of character for Reggie.